website/videoBlog/2024-02-03-Apache-Hudi-Table-Services-Export-Services-HoodieSnapshotExporter-Hands-on-labs.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "Apache Hudi Table Services | Export Services | HoodieSnapshotExporter | Hands on labs"
+last_modified_at: 2024-02-21T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-02-03-Apache-Hudi-Table-Services-Export-Services-HoodieSnapshotExporter-Hands-on-labs.png
+navigate: "https://www.youtube.com/watch?v=r2-N-oF5QIE"
+tags:
+- guide
+- beginner
+- apache hudi
+- HoodieSnapshotExporter
```

Comment:
This seems like a java class name. Let's break this into separate words for tagging purposes -> `hudi snaphot exporter`